src/fcio.h

```diff
@@ -226,7 +226,11 @@ typedef enum {
   FCIOEvent = 3,
   FCIOStatus = 4,
   FCIORecEvent = 5,
-  FCIOSparseEvent = 6
+  FCIOSparseEvent = 6,
+  FCIOEventHeader = 7,
+  FCIOFSPConfig = 8,
```

Comment:
Perhaps add some comment that these three tags are reserved for the Fast Stream Processor.

Reply:
Addressed